packages/cli/src/Server.ts

```diff
@@ -388,6 +394,7 @@ class App {
 		// refresh enterprise status
 		Object.assign(this.frontendSettings.enterprise, {
 			sharing: isSharingEnabled(),
+			logStreaming: isLogStreamingEnabled(),
```

Comment:
Could you please add this setting to the `IN8nUISettings` interface in `packages/cli/src/Interfaces.ts`?